tools/sggolicenses/tools.go

```diff
@@ -25,6 +25,38 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 	return sg.Command(ctx, sg.FromBinDir(name), args...)
 }
 
+// Check for disallowed types of Go licenses in a specific directory.
+// By default, Google's forbidden and restricted types are disallowed.
+func CheckInDirectory(ctx context.Context, directory string, disallowedTypes ...string) error {
```

Comment:
since we use `Dir` in other parts of Sage (as the `sg.FromBinDir` above) let's call it `CheckDir`?

Reply:
Sure

Reply:
Fixed it